1.0/files/docker-entrypoint.sh

```diff
@@ -76,6 +76,12 @@ if [[ $GENERATE_NEW_SAVE == true ]]; then
     fi
 fi
 
+# check to see if the user has specified a custom port environment variable
+if [[ ! -z "$CUSTOM_PORT" && "$CUSTOM_PORT" -eq "$CUSTOM_PORT" ]]; then
```

Comment:
```suggestion
if [[ ! -z "$CUSTOM_PORT" && "$CUSTOM_PORT" -eq "$CUSTOM_PORT" ]]; then
```
Why is the second part necessary?

Reply:
Hi, this is just a safety check to confirm that the CUSTOM_PORT is actually set and is indeed a number.

Reply:
We don't do this anywhere else and just require admins to correctly set variables. I don't think this is necessary.

Reply:
```suggestion
if [[ -n $CUSTOM_PORT ]]; then
```

Reply:
Yeah, cool that's works too